runners/google-cloud-dataflow-java/worker/windmill/src/main/proto/windmill.proto

```diff
@@ -446,6 +446,8 @@ message StreamingCommitWorkRequest {
 message JobHeader {
   optional string job_id = 1;
   optional string project_id = 2;
+  // Worker id is meant for logging only. Do not rely on it for other decisions.
```

Comment:
Hah. 5 mins later, being used for some mission critical thing.

Reply:
Hmm... in a design discussion?